src/api-doc/Entity Management API's.postman_collection.json

```diff
@@ -0,0 +1,977 @@
+{
+	"info": {
+		"_postman_id": "fa7d1558-cdcb-493a-8767-55758ac735da",
+		"name": "Entity Management API's",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "33513401"
+	},
+	"item": [
+		{
+			"name": "entity",
+			"item": [
+				{
+					"name": "Find",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "x-auth-token",
+								"value": "{{userToken}}",
+								"type": "text"
+							},
+							{
+								"key": "content-type",
+								"value": "application/json",
+								"type": "text"
+							},
+							{
+								"key": "internal-access-token",
+								"value": "{{internalToken}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"query\": {\n        \"name\": \"Goa\"\n    },\n    \"projection\": {\n        \"immediateChildrenEntityType\": 1,\n        \"name\": 1\n    },\n    \"skipFields\": {\n        \"createdAt\": 1\n    }\n}\n",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{baseUrl}}/entity/v1/entityTypes/find",
```

Comment:
@MallanagoudaB {{baseUrl}}/entity/ will not work right it should be {{baseUrl}}/entity-management/
Please check all the urls in this postman collection. Very Important! not repeating same comment  again again